server/routes/auth.js

```diff
@@ -50,23 +50,9 @@ router.post('/signup', async (req, res) => {
 
         // Create a new user in the database
         //for mock data, have all users start out with location at meta's headquarters
-        const newUser = await prisma.user.create({
-            data: {
-                address,
-                username,
-                latitude: 37.4855,
-                longitude: -122.1500,
-                email,
-                password: hashedPassword,
-                events: {
-                    create: [
-                        { event: { create: { title: `${username}_bowling`, description: 'descBowling', address: 'event avenue', zip_code: '75034', latitude: 37.4855, longitude: -122.1500 } } },
-                        { event: { create: { title: `${username}_tennis`, description: 'descTennis', address: 'event avenue', zip_code: '75034', latitude: 37.4855, longitude: -122.1500  } } },
-                    ],
-                },
-            }
-        })
+        const queryResult = await prisma.$queryRaw`INSERT INTO "User" (address, username, email, password, coord) VALUES(${address}, ${username}, ${email}, ${hashedPassword}, ST_SetSRID(ST_MakePoint(${-122.1500}, ${37.4855}), 4326)::geography) RETURNING id, address, username, email, password, ST_AsText(coord);`;
 
+        const newUser = queryResult[0];
```

Comment:
Consider [Array.at](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) for index safety